ksqldb-test-util/src/main/resources/log4j.properties

```diff
@@ -20,6 +20,7 @@ log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 # help debug a flaky test
 log4j.logger.io.confluent.ksql.api.server=DEBUG
+log4j.logger.io.confluent.rest.server.HeartbeatAgent=DEBUG
```

Comment:
I was going to add this in a different PR to help debug a flaky test but forgot to create a branch and it snuck in. hopefully not a problem

Reply:
sounds good